Sources/MongoSwift/BSON/BSONEncoder.swift

```diff
@@ -102,7 +110,25 @@ public class BSONEncoder {
     }
 
     /// Initializes `self`.
-    public init() {}
+    public init(options: CodingStrategyProvider? = nil) {
+        self.configureWithOptions(options: options)
+    }
+
+    /// Initializes `self` by copying the options of another `BSONEncoder`.
```

Comment:
same comment about clarifying the behavior around conflicting options on `other` and `options`.

Reply:
done